vnext/Microsoft.ReactNative.Cxx.UnitTests/Microsoft.ReactNative.Cxx.UnitTests.vcxproj

```diff
@@ -66,7 +66,7 @@
       <ForcedIncludeFiles>pch.h</ForcedIncludeFiles>
       <PreprocessorDefinitions>_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <WarningLevel>Level4</WarningLevel>
-      <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory);$(JSI_SourcePath);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
>$(JSI_SourcePath) [](start = 64, length = 17)

OOC, where is that value coming from?

Reply:
It is coming from React.Cpp.props and can be redrected to the patched files folder. To enable the use in VS IDE, I am providing a default location in the iteration 6.

---
In reply to: [451863266](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r451863266) [](ancestors = 451863266)